rudder-core/src/main/scala/com/normation/rudder/api/ApiAccountRepository.scala

```diff
@@ -65,11 +67,11 @@ trait RoApiAccountRepository {
   /**
    * Retrieve all API Account
    */
```

Comment:
I don't think the API should have that boolean. In generale, when we did that, it was error prone and a pain to use and upgrade. 
More over, AFAIK, there is only one place where we need to check for system token (in rest auth). And perhaps we want to do a special log for that one, to more specificaly trace the use of that token (what client used it for ex - we will to do that kind of log at some point). 

So, I would prefer to let the API as it is for existing method, + add a "getSystemToken" (ou Tokens, not sure we plan to add more, but for now, singular seems sufficient) and update the authentication method to call both.